clang/lib/Sema/SemaX86.cpp

```diff
@@ -162,6 +162,9 @@ bool SemaX86::CheckBuiltinRoundingOrSAE(unsigned BuiltinID, CallExpr *TheCall) {
   case X86::BI__builtin_ia32_mulps512:
   case X86::BI__builtin_ia32_subpd512:
   case X86::BI__builtin_ia32_subps512:
+  case X86::BI__builtin_ia32_vaddpd256_round:
```

Comment:
missing error check test?

Reply:
I don't think it's necessary. For one thing, it's verbose given we have 149 new rounding instructions. For another, we didn't add test for BI__builtin_ia32_addpd512 either.